imagery/i.atcorr/create_iwave.py

```diff
@@ -178,25 +168,25 @@ def pretty_print(filter_f):
     Create pretty string out of filter function
     8 values per line, with spaces, commas and all the rest
     """
-    pstring = ""
+    pstring = []
     for i in range(len(filter_f) + 1):
-        if i % 8 is 0:
-            if i is not 0:
+        if i % 8 == 0:
```

Comment:
Just a infinitesimal implementation-dependent optimisation possible here: I stumbled upon something on stack overflow this weekend, not exactly related to this, but the example had this form.

This pattern uses at least 2 more opcodes than another possibility, as it has to protect himself from the variable not being an integer when comparing to 0. There's an alternative, but it was open three days ago on my computer, and I'm writing on my phone. It had something to do with either using "in" or nothing at all.

I'm just nerdsniping here, if you are curious to microoptimise here.

Reply:
Would be nice to microoptimise but I am not sure if it is worth using `is` which is intended for booleans and `None`.